Examples-Web/Depth/Depth.css

```diff
@@ -0,0 +1,40 @@
+body {
+    --zoom: 180%;
+    height: 100vh;
+    width: 100vw;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden;
+    background-color: #073547;
+}
+
+#container {
+    height: min(100vh, 100vw);
+    width: min(100vh, 100vw);
+    display: grid;
+    grid-template-rows: var(--zoom);
+    grid-template-columns: var(--zoom);
```

Comment:
💬 Would flex be easier here? No need to defined rows and cols that way

Reply:
I tried flex first but it was easier to have a single cell in the grid with expands centrally.

I can look more into it if you don't like this solution 🤷